src/java.base/share/classes/java/text/MessageFormat.java

```diff
@@ -553,6 +553,9 @@ private void applyPatternImpl(String pattern) {
      * The string is constructed from internal information and therefore
      * does not necessarily equal the previously applied pattern.
      *
+     * @implNote The string returned by this method can be used to create
```

Comment:
Hmm, I'm not sure about the current note, because its not true in all cases (for example, some unknown subclass of Format). Maybe @naotoj has thoughts?

Reply:
Yes, my comment to the CSR was to explain the default implementation as the `@implNote`. Making it explicit would be helpful to readers.

Reply:
Yes my mistake, I'll change this to:
```
@implNote The implementation in {@link MessageFormat} returns a string
that can be used to create a new instance that is semantically equivalent
to this instance.
```